### DISCUSSION TITLE
Control the temp for blue-light-filter?

### DISCUSSION THREAD

**Discussion by semi710** [loqusion/hyprshade#27] [Q&A] [answered]
Didn't find in documentation how can I change the temprature?

**Comment by loqusion** [comment, marked as answer]
In v4.0.0 on, shaders will be configurable in `hyprshade.toml` (like in [this discussion](https://github.com/loqusion/hyprshade/discussions/25#discussioncomment-8780367)).

For now, you can copy [`blue-light-filter.glsl`](https://github.com/loqusion/hyprshade/blob/85e7384bd8483e56ad48714696c098741d47c1b5/shaders/blue-light-filter.glsl#L8) to `~/.config/hypr/shaders` (or `~/.config/hyprshade/shaders`) and modify `temperatureStrength`.

**Reply by nullbyto** [reply]
Is it now possible to set in `hyprshade.toml` ? If yes how ?
